### ISSUE TITLE
Add typescript support

### ISSUE THREAD

**Issue by amokrunnr** [clinggroup/vite-plugin-rollbar#7]
Please add typescript declarations. Thanks!

**Comment by amokrunnr** [comment]
Here is the diff that solved my problem:

```diff
diff --git a/node_modules/vite-plugin-rollbar/index.d.ts b/node_modules/vite-plugin-rollbar/index.d.ts
new file mode 100644
index 0000000..ab208d8
--- /dev/null
+++ b/node_modules/vite-plugin-rollbar/index.d.ts
@@ -0,0 +1,16 @@
+import { Plugin } from 'vite';
+
+type Options = {
+  accessToken:string
+  version:string
+  baseUrl:string
+  silent?:boolean,
+  rollbarEndpoint?:string,
+  ignoreUploadErrors?:boolean
+  base?:string
+  outputDir?:string
+}
+
+declare function plugin(options:Options): Plugin;
+
+export {  plugin as default };
diff --git a/node_modules/vite-plugin-rollbar/package.json b/node_modules/vite-plugin-rollbar/package.json
index 290e733..0e7e7e6 100644
--- a/node_modules/vite-plugin-rollbar/package.json
+++ b/node_modules/vite-plugin-rollbar/package.json
@@ -7,6 +7,7 @@
   "module": "./esm/index.js",
   "exports": {
     ".": {
+      "types": "./index.d.ts",
       "require": "./cjs/index.js",
       "default": "./esm/index.js"
     }
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by a0s** [comment]
Hello, i found this issue too..

**Comment by rang-ali** [comment]
Fixed #9

**Comment by muhammad-sayed-mahdy** [comment]
why this is not released yet?
